src/__tests__/createStore.test.js

```diff
@@ -13,7 +13,25 @@ const storeWithIgnore = storeFactory({ ignore: ['bar', 'baz'] })
 const storeWithOnly = storeFactory({ only: ['foo'] })
 const storeWithConflicts = storeFactory({ only: ['bar'], ignore: ['bar'] })
 
-beforeEach(() => localStorage.clear())
+const errorHandler = jest.fn()
+const errorStoreAdapter = {
+  get(key) { throw Error('Error getting key') },
+  set(key, value) { throw Error('Error setting key') },
+  clear() {throw Error('Error clearing storage') },
+  remove(key) { throw Error('Error removing key') },
+  afterGet() {},
+  beforeSet(data) { return data },
+  onGetError(error, key) { errorHandler(error, key) },
```

Comment:
Can you replace `error` with `err` please? This would remove any confusion with the `Error` constructor.